### PR TITLE
Initial configuration of End-to-end automation tests

### PR DESCRIPTION
**Pull request by cp4r3z**
## Summary

Relating to #609, this initial test simply opens the landing page for a new user and hits the login button. This is sufficient to know that jest is playing nicely with Selenium and not interfering with the existing unit tests.

## Developer notes

The development guidelines didn't specify, but let me know if you prefer squashed commits. Personally, I prefer small and meaningful commits for more specific annotation (blame)

### CODE REVIEW

**Comment by shakalee14** [comment]
@cp4r3z 
1) A commit history that's clear and includes meaningful changes is best
2) Also, note: If a pr isn't ready for review, please tag it with `wip`; if it doesn't have a `wip` tag, I can assume it's ready to be reviewed.

File: docs/HOWTO-run_tests.md [hunk 1]
```diff
@@ -14,3 +14,21 @@ GRANT ALL PRIVILEGES ON DATABASE spoke_test TO spoke_test;
 ## SQLite Testing (simpler)
 
 1) Run `npm run test-sqlite`
+
+## End-To-End (Interactive Browser) Testing
+
+1. Remember to set `NODE_ENV=dev` 
+1. Setup DB
```
Comment (shakalee14) [inline docs/HOWTO-run_tests.md:21]: When you mean setup DB, you mean like whats apart of the startup guide? Let's try to be  as clear as possible.

File: docs/HOWTO-run_tests.md [hunk 2]
```diff
@@ -14,3 +14,21 @@ GRANT ALL PRIVILEGES ON DATABASE spoke_test TO spoke_test;
...
+1. Start Spoke Server
```
Comment (shakalee14) [inline docs/HOWTO-run_tests.md:22]: Can we put a sample script here to run, like `npm run dev`

**Review by shakalee14 — COMMENTED** [review]
Overall - LGTM - still reviewing but I can get this to run locally.

File: docs/HOWTO-run_tests.md (hunk 2 above)
Comment (cp4r3z) [reply docs/HOWTO-run_tests.md:22]: Yes, the next PR will have the "final" script which will be `npm run test-e2e`

File: docs/HOWTO-run_tests.md (hunk 1 above)
Comment (cp4r3z) [reply docs/HOWTO-run_tests.md:21]: At the time of this PR I still wasn't sure what "SetupDB" meant. I've added a note to myself to elaborate.

**Comment by shakalee14** [comment]
Hey @cp4r3z - are you planning on updating this PR or including this PR and requested changes in the next one?

**Comment by cp4r3z** [comment]
Changes will be in the next PR (still in progress)

**Pull request merged**